frontend/src/scenes/dashboard/DashboardItem.tsx

```diff
@@ -84,8 +85,8 @@ export const displayMap: Record<DisplayedType, DisplayProps> = {
         className: 'graph',
         element: ActionsLineGraph,
         viewText: 'View graph',
-        link: ({ filters, id, dashboard, name }: DashboardItemType): string =>
-            combineUrl(urls.insightView(id, filters), undefined, {
+        link: ({ filters, short_id, dashboard, name }: DashboardItemType): string =>
```

Comment:
nit: I think it might not be worth having this link key here if the values are all the same.

Reply:
Cleaned this up.